projects/inih/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update
+RUN git clone https://github.com/0x34d/inih
```

Comment:
I will make this change after https://github.com/benhoyt/inih/pull/153 is accepted.
Then, you can merge.

Reply:
Sounds good. I will double-check if the panel is interested in integrating `inih`.

Reply:
@Alan32Liu ping